packages/server/utils/analytics/analytics.ts

```diff
@@ -126,9 +132,27 @@ export type AnalyticsEvent =
   | 'Suggested Groups Generated'
   | 'Suggest Groups Clicked'
   | 'Reset Groups Clicked'
+  // Conversion Tracking
+  | 'Conversion Modal Pay Later Clicked'
+  // Deprecated Events
+  // These will be replaced with tracking plan compliant versions by the data team
+  | 'Added Agenda Item'
+  | 'Archive Organization'
+  | 'Archive Team'
+  | 'Enterprise Over User Limit'
+  | 'Locked user attempted to join a team'
+  | 'Mattermost notification sent'
+  | 'Mentioned on Task'
+  | 'MSTeams notification sent'
+  | 'New Org'
+  | 'New Team'
+  | 'Poll added'
+  | 'Slack notification sent'
+  | 'Smart group title changed'
+  | 'Task due date set'
```

Comment:
Technically you can apply the renaming trick I mentioned below to clean these up but understandable it can wait

Reply:
My plan was to handle bringing the events into compliance later once we start to improve our tracking plan.